index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                                <li> <a href="https://arxiv.org/pdf/1707.06347"> Proximal
+                                        Policy Optimization.</a> Schulman et al. (2017)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Wed, April 16
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Q-Learning</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i> <b>Survey Due</b><br>
+                        </td>
+                        <td>
+
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/1312.5602"> Playing
+                                        Atari with Deep Reinforcement Learning.</a> Mnih et al. (2013)</li>
+                            </ul>
+
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Fri, April 18
+                        </td>
+                        <td><i><span class="label label-lecture text-base">TA Session</span></i>
+                            <a">Extra section on Q-learning</a>
+                        </td>
+                        <td></td>
+                        <td>
+                            <ul>
+                                <li> Logistics: 1:30 - 2:50 pm in Gates B01</li>
+                                <li> <a href="https://arxiv.org/abs/1509.06461"> Learning to Predict by the Methods of
+                                        Temporal Differences </a> Sutton (1988)</li>
+                                <li> <a href="https://arxiv.org/abs/1806.10293"> Online Q-Learning using Connectionist
+                                        Systems
+                                    </a> Rummery et al. (1994)
+                                </li>
+                                <li> <a href="https://arxiv.org/abs/1806.10293"> Flow Q-Learning
+                                    </a> Park et al. (2025)
+                                </li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Fri, April 18
+                        </td>
+                        <td><i><span class="label label-lecture text-base"> Lecture</span></i>
+                            <a">Practical Deep RL Implementation
+                                Techniques</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i><b> HW1 Due</b><br>
```

Comment:
Note, here and elsewhere you don't need the extra "Due" text because of the label.